app/models/cda/court_application.rb

```diff
@@ -3,6 +3,12 @@ class CourtApplication < BaseModel
     has_one :subject_summary, class_name: 'Cda::SubjectSummary'
     has_many :hearing_summary, class_name: 'Cda::ApplicationHearing'
 
+    def self.from_urn(urn)
```

Comment:
since we have `CourtApplication.find`
maybe call this 
`CourtApplication.find_from_urn` for consistency?